### ISSUE TITLE
The input makes auto delete the value

### ISSUE THREAD

**Issue by jxnx888** [bl00mber/react-phone-input-2#447]
Hello Developer,

There is a bug I cannot resolve from my end. Please take a look.

If I am using the Chinese input method or other's input method which will popup the options, the input will automatically delete when I press the key 'space'. And also it appears on the iPad.

Attach is my screenshot.
![autoDelete](https://user-images.githubusercontent.com/22604980/128312461-e832d174-ad0d-46bc-9a80-bba11073da87.gif)

https://user-images.githubusercontent.com/22604980/128312845-4ef98761-05e3-4558-8105-835464e963b6.mp4

**Comment by nfl404** [comment]
I have the same problem.

**Comment by XiaoBieThree** [comment]
Right, I have also encountered this situation. When the input method pops up at the pad end, this event will also be triggered if you enter non English, and the last bit of the value entered in the input box will be deleted automatically
